src/sentinel.c

```diff
@@ -2776,7 +2776,7 @@ void sentinelInfoReplyCallback(redisAsyncContext *c, void *reply, void *privdata
     link->pending_commands--;
     r = reply;
 
-    if (r->type == REDIS_REPLY_STRING)
+    if (r->type == REDIS_REPLY_STRING || r->type == REDIS_REPLY_VERB)
```

Comment:
"info" return type is verbatim in resp3

Reply:
I ran `runtest-sentinel` with "client-default-resp 3" config and realized we need these two fixes (see the similar one below). So, it seems like sentinel won't work with resp3 replies currently. 

I'm not sure if we are okay with this fix or do we want a separate PR for it or should we solve this issue some other way? Please comment.

Reply:
i'm not very good with sentinel, so let me see if i understand this correctly.
this change doesn't change how sentinel responds to clients, but rather how sentinel understands replies it gets from redis when redis speaks RESP3, right?

in theory, i'd rather not do other changes (bug fixes) in the reply schema PR, i prefer to take them into separate PRs and mention the changes in the release notes.

however, i'm guessing that normally sentinel will never send `HELLO 3` to redis, so it will never get a RESP3 reply, and this bug is surfaced now just because we changed RESP3 to be the default in redis.
@moticless please ack.

if that's right, then it's completely internal to the test changes in our reply schema PR, and can be put there.
we should mention it in the top comment of the PR just in order to avoid confusion for anyone reading the code.

Reply:
Yes, your understanding is correct.

Reply:
@oranagra , indeed, sentinel doesn't send `HELLO 3` .

Reply:
@tezc please add a comment in both places ("Sentinel will never send HELLO 3 except in tests where we use --force-resp3 --log-req-res (see logreqres.c)")

maybe even
```
#ifdef LOG_REQ_RES
int valid = r->type == REDIS_REPLY_ARRAY;
#else
int valid = r->type == REDIS_REPLY_STRING || r->type == REDIS_REPLY_VERB;
#endif

if (valid) {
...
```

same for the other place 

@oranagra fine with you?

Reply:
>  "Sentinel will never send HELLO 3 except in tests where we use --force-resp3 --log-req-res (see logreqres.c)"

This is not so accurate. The problem is when we switch to resp3 by default (by using `client-default-resp 3` config), currently sentinel cannot parse resp3 replies. (sentinel never sends hello 3 itself in any case, just the default protocol becomes resp3). 

So, it is a bug in sentinel. I'm leaving the decision to you whether we want to make sentinel able to work with above config or move the fix inside an ifdef for the test only.

Reply:
i feel like we don't want to fix any "bugs" in sentinel

if one day someone decides to make sentinel work with RESP3, then we can delete the ifdef

Reply:
redis will never switch to RESP3 by default, and since sentinel never sends HELLO 3, this isn't a bug.
i think an ifdef is excessive (adds more lines), it's enough to add one line of comment IMHO

Reply:
added a comment